webapp/components/Table/Content/Content.js

```diff
@@ -24,7 +25,7 @@ const Content = (props) => {
   ) : (
     <div className="table__content">
       <div className="table__row-header" style={{ gridTemplateColumns }}>
-        {React.createElement(rowHeaderComponent, props)}
+        {React.createElement(rowHeaderComponent, { props, ...otherProps })}
```

Comment:
so you really need to speed props here?
is so, can they be called rowProps ?

Reply:
ok